R/calc_plan_stats.R

```diff
@@ -0,0 +1,88 @@
+calc_plan_stats <- function(redist_plans, redist_map, ...) {
+
+    redist_plans <- redist_plans %>%
+        mutate(total_vap = tally_var(redist_map, vap),
+               plan_dev =  plan_parity(redist_map),
+               comp_edge = distr_compactness(redist_map),
+               comp_polsby = distr_compactness(redist_map,
+                                               measure = "PolsbyPopper"),
+               ndv = tally_var(redist_map, ndv),
+               nrv = tally_var(redist_map, nrv),
+               ndshare = ndv / (ndv + nrv),
+               ...)
+
+    tally_cols <- names(redist_map)[c(tidyselect::eval_select(starts_with("pop_"), redist_map),
```

Comment:
may need `tidyselect::starts_with()`

You can test locally by running `devtools::check()` or (better) going to the 'Build' tab in the upper right of Rstudio and pressing "check"